lib/cve_server/app.rb

```diff
@@ -22,6 +22,21 @@ class App < Sinatra::Base
       end
     end
 
+    get '/v1/cves/:cves' do |cves|
+      answer = []
+      elements = cves.split(',').uniq
+      bad_request if elements.empty? || elements.size > 10
+      elements.each do |e|
+              e.match(/^CVE-\d{4}-(0\d{3}|[1-9]\d{3,})$/) do |match|
+                cve = match[0]
+                bad_request unless valid_cve?(cve)
+                cve = CVEServer::Cve.find(cve.upcase)
+                answer.append(cve) if cve
+              end
+          end
+      json_resp answer
```

Comment:
It also could be good idea to rename the answer variable to something like entries or records.

Reply:
You are right. Thanks!

Addressed in 9047ab5